org.kicad_pcb.KiCad.json

```diff
@@ -0,0 +1,130 @@
+{
+    "app-id": "org.kicad_pcb.KiCad",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "kicad",
+    "appdata-license": "GPL-2.0+ and BSD and MIT and wxWidgets",
+    "rename-desktop-file": "kicad.desktop",
+    "rename-icon": "kicad",
+    "rename-appdata-file": "kicad.appdata.xml",
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--filesystem=home"
+    ],
+    "cleanup": ["/share/doc",
+        "/share/aclocal", "/share/bakefile",
+        "/lib/wx", "/include",
+        "*.la", "*.a"],
+    "modules": [
+        {
+            "name": "libglu",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://mesa.freedesktop.org/archive/glu/glu-9.0.0.tar.bz2",
+                    "sha256": "1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12"
+                }
+            ]
+        },
+        {
+            "name": "GLEW",
+            "no-autogen": true,
+            "make-args": [
+                "GLEW_PREFIX=/app",
+                "GLEW_DEST=/app",
+                "LIBDIR=/app/lib"
+            ],
+            "make-install-args": [
+                "GLEW_PREFIX=/app",
+                "GLEW_DEST=/app",
+                "LIBDIR=/app/lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/project/glew/glew/2.1.0/glew-2.1.0.tgz",
+                    "sha256": "04de91e7e6763039bc11940095cd9c7f880baba82196a7765f727ac05a993c95"
+                }
+            ]
+        },
+        {
+            "name": "wxWidgets",
```

Comment:
experience here is that you're going to need to force a reconfigure to make this understand Aarch64. See examples [in flathub](https://github.com/search?q=org%3Aflathub+wxwidgets&type=Code).